lesson12/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
nits
The language written is Japanese.
```suggestion
<html lang="ja">
```

Reply:
I fixed!
I didn't notice until now.
-> 6c6a888